src/main/java/rx/internal/operators/OperatorAll.java

```diff
@@ -61,7 +61,10 @@ public void onNext(T t) {
 
             @Override
             public void onError(Throwable e) {
-                child.onError(e);
+                if (!done) {
+                    done = true;
+                    child.onError(e);
+                }
```

Comment:
Since you pointed it out in both PRs would not this also apply to some other places?